src/javis_viewer.jl

```diff
@@ -0,0 +1,292 @@
+# TODO: Intelligent way of handling slider
+
+function javis_viewer(; video::Video, frames::Int, action_list::Vector)
```

Comment:
The man who loves keyword arguments really uses them in weird places.
This will completely fail if you call `javis_viewer()` I suppose.

Reply:
This is quite true... I turned this into a dunder function. I also like the nickname of "the man who loves keyword arguments".